.changeset/proud-cows-own.md

```diff
@@ -0,0 +1,7 @@
+---
+"@commercetools-frontend/babel-preset-mc-app": patch
+---
+
+Set Node.js version to `current` in the `babel-preset-mc-app` for `babel-preset-env`
+
+The previous version was hard-coded to `8` while `current` whill always use the version of the running process.
```

Comment:
This is only valid for the test environment, right?

Reply:
Yes, it's within a block of `isEnvTest`. If means less transpilation for Node.js as otherwise we transpile for 8 to run in 16 which is wasteful.